bridge/client/app/_components/ktb-project-settings/ktb-project-settings.component.ts

```diff
@@ -61,9 +64,10 @@ export class KtbProjectSettingsComponent implements OnInit, OnDestroy, PendingCh
   public unsavedDialogState: DialogState = null;
   public resourceServiceEnabled?: boolean;
 
-  readonly keptnInfo$ = this.dataService.keptnInfo.pipe(
+  readonly state$: Observable<ProjectSettingsState> = this.dataService.keptnInfo.pipe(
     filter((keptnInfo: KeptnInfo | undefined): keptnInfo is KeptnInfo => !!keptnInfo),
-    takeUntil(this.unsubscribe$)
+    map((keptnInfo: KeptnInfo) => ({ gitUpstreamRequired: !keptnInfo.metadata.automaticprovisioning })),
+    startWith({ gitUpstreamRequired: true })
```

Comment:
Technically this could lead to incorrect behavior. If the `keptnInfo` observable returns after the feature flag for the resource service is set, the `ngOnInit` in `bridge/client/app/_components/ktb-project-settings-git-extended/ktb-project-settings-git-extended.component.ts` would not select the correct form, because the `ngOnInit` is triggered only once with the `startWith` value instead of the right one.

Reply:
The `ktb-project-settings-git-extended` component only shows after `isProjectLoading` is true. Would this still be a problem?

Reply:
Yes, because the project/projects call may return before the `keptnInfo`. But I just realized that the featureFlag should not return before the keptnInfo because it depends on it.